docs/attestors/jwt.json

```diff
@@ -243,6 +243,30 @@
             "$ref": "#/$defs/OID"
           },
           "type": "array"
+        },
+        "InhibitAnyPolicy": {
+          "type": "integer"
+        },
+        "InhibitAnyPolicyZero": {
+          "type": "boolean"
+        },
+        "InhibitPolicyMapping": {
+          "type": "integer"
+        },
+        "InhibitPolicyMappingZero": {
+          "type": "boolean"
+        },
+        "RequireExplicitPolicy": {
+          "type": "integer"
+        },
+        "RequireExplicitPolicyZero": {
+          "type": "boolean"
+        },
+        "PolicyMappings": {
+          "items": {
+            "$ref": "#/$defs/PolicyMapping"
+          },
+          "type": "array"
```

Comment:
It looks like the `jwt` attestor and everything that embedded it changed. Should we bump the version on all those attestor types?

Reply:
Let me look into this real quick. I'm unsure of the changes to the JWT attestor and if they should have happened.

Reply:
Hm.. okay so what happened here is we're embedding a `jose.JSONWebKey` in the JWT attestor, which contains an array `x509.Certificate` in the struct. 

Honestly, it is probably a mistake for the jwt attestor to use a third-party struct as part of it's schema.

Reply:
Let's skip a version bump on the attestors in this case. In this particular case, I feel confident there's little to no risk of these schema changes having an impact.

However, before witness 0.9 is released, proper attestation versioning must be sorted. And I think we will need to re-work the JWT attestor a bit and do a true v0.2 of that attestor.